internal/venice-client-common/src/main/java/com/linkedin/venice/compression/NoopCompressor.java

```diff
@@ -16,7 +16,7 @@ public byte[] compress(byte[] data) throws IOException {
   }
 
   @Override
-  public ByteBuffer compress(ByteBuffer data) throws IOException {
+  public ByteBuffer compress(ByteBuffer data, int startPositionOfOutput) throws IOException {
```

Comment:
Hmm, yes, good point, that's a miss... I wonder why this doesn't fail any test though... I need to investigate this...

Reply:
So I've added some defensive code here... in case a future refactoring makes us use this function when for NoOp... please take a look and LMK if you have any concerns.